DependencyInjection/TrikoderOAuth2Extension.php

```diff
@@ -70,6 +71,9 @@ public function load(array $configs, ContainerBuilder $container)
                 'method' => 'onKernelException',
                 'priority' => $config['exception_event_listener_priority'],
             ]);
+
+        $container->registerForAutoconfiguration(GrantTypeInterface::class)
+            ->addTag('authorization_server.grant');
```

Comment:
@X-Coder264 @spideyfusion I think we should prefix this with `trikoder.oauth2.`, WDYT?

Reply:
@HypeMC @X-Coder264 Agreed.